### DISCUSSION TITLE
How to turn a html site into a php site?

### DISCUSSION THREAD

**Discussion by gingerbeardman** [EasyEngine/easyengine#1868] [Support Requests] [answered]
is there a recommended way to turn a html site into a php site?

- update does not work
- sync and clone only work for WordPress sites

```
ee site update myexample.com --php='7.4'
Starting php version update for: myexample.com
Taking backup of old php config.
Error: The origin directory specified "/opt/easyengine/sites/myexample.com/config/php" was not found.
```

**Comment by gingerbeardman** [comment, marked as answer]
My workaround, which was less than ideal, but not too much effort:

1. create temp site to backup some files
    - `ee site create backup.example.com --php=7.4`
2. copy files manually on command line from html site to temp site
    - `cp -R /opt/easyengine/sites/html.example.com/app/htdocs/* /opt/easyengine/sites/backup.example.com/app/htdocs/`
3. delete html site (had SSL)
    - `ee site delete html.example.com`
6. create new php site (with SSL)
    - `ee site create php.example.com --php=7.4 --ssl=le`
7. copy files manually on command line from temp site to php site
    - `cp -R /opt/easyengine/sites/backup.example.com/app/htdocs/* /opt/easyengine/sites/php.example.com/app/htdocs/`

Downtime was only a few minutes.

**Reply by marcoandrei** [reply]
Hello, @gingerbeardman!

Maybe my reply is a little late, but currently you can't change the type of the site using `site update` as far as I know. You only can add things.

So you need to backup your site, delete and create again using `--type=php`. 

Well, as you already did. :-) I had this problem in the past too.

Kind regards,
Marco Andrei

**Reply by gingerbeardman** [reply]
No worries! Since a few months ago I've migrated all my sites to Caddy server.

**Reply by marcoandrei** [reply]
Oh, nice! Well, I think this conversation can be marked as Answered, right? :-)

I don't know Caddy, but it catched my attention, I will take a look soon for sure!

So, happy new year, @gingerbeardman! Saúde!

**Reply by gingerbeardman** [reply]
Right on!

Season's Greetings to you and yours
